app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -401,24 +410,34 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                               props.setFieldValue('licenseId', source.licenseId);
                             }
                           }}
-                          options={sourceOptions}
+                          options={sourceOptions.filter(
+                            (x) =>
+                              !isImageForm(contentType) ||
+                              x.label.includes('(TC)') ||
```

Comment:
What is this for?  Why would we choose to hardcode these values?

Reply:
Based on the business for the front pages form we only need to show 5 source options as above. Thanks